### ISSUE TITLE
Refactor elastic client out of services, move to Satellite

### ISSUE THREAD

**Issue by humphd** [Seneca-CDOT/telescope#2134]
We have a few places in our microservices where we need to connect to Elasticsearch:

- Search service: src/api/search/src/elastic.js
- Parser service: src/api/parser/src/lib/elastic.js

Let's move this to Satellite, similar to [what we did with Redis](https://github.com/Seneca-CDOT/satellite/blob/master/src/redis.js).  The goal would be for microservices to be able to do this:

```js
const { Elastic } = require('@senecacdot/satellite');

const elasticClient = Elastic();
```

NOTE: I also think the Parser service is missing the `indexer`, which it needs to talk to Elastic (cc @c3ho).

**Comment by HyperTHD** [comment]
Taking this, will try and have a PR up by tomorrow's triage meeting

**Comment by humphd** [comment]
This has landed in Satellite.  I've triggered a [new release](https://github.com/Seneca-CDOT/satellite/actions/runs/761624069), and when it completes, this will be available in v1.13.0.
